projectorganizer/src/prjorg-project.c

```diff
@@ -276,13 +276,24 @@ static GeanyFiletype *filetypes_detect(const gchar *utf8_filename)
 }
 
 
+static gboolean string_exists_in_array(gchar **array, gchar *str) {
```

Comment:
I'm pretty sure this function already exists in glib or somewhere... I just couldn't find it :roll_eyes:

Reply:
> I'm pretty sure this function already exists in glib or somewhere...

Perhaps you mean [g_strv_contains](https://developer-old.gnome.org/glib/stable/glib-String-Utility-Functions.html#g-strv-contains)?